helpers/clean-kibana-indexes.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Description: script to remove old .kibana* indexes' aliases created after Kibana
+# restart.
+#
+# The aim of the script is to keep only the latest index with the latest 
+# configuration, and delete the old ones.
+
+# Usage is: 
+# CLUSTER="amazeeio-test" ELASTICSEARCH_NAMESPACE="elasticsearch" ./clean-kibana-indexes.sh"
+# CLUSTER="amazeeio-test" ELASTICSEARCH_NAMESPACE="elasticsearch" TENANT="drupalexample" ./clean-kibana-indexes.sh"
+
+set -eu
+
+# Set DEBUG variable to true, to start bash in debug mode
+DEBUG="${DEBUG:-"false"}"
+if [ "$DEBUG" = "true" ]; then
+        set -x
+fi
+
+# Cluster to connect to
+CLUSTER="${CLUSTER:-""}"
```

Comment:
`CLUSTER` is quite confusing here, the cluster of what? where the tenant has their projects? or where the elasticsearch is located? let's call this something better

Reply:
Kubernetes/Openshift cluster where ES is located.
I changed "Target Kubernetes/Openshift cluster where Elasticsearch is located"
I use it to switch kubectl context

Reply:
please change the actual variable to make it more clearer, not just the comment.